.github/workflows/build-images.yml

```
@@ -36,7 +36,7 @@ jobs:
           make O=.output LOCALVERSION= defconfig sdm660_defconfig
       - name: Build linux kernel (Image.gz)
         run: |
-          make O=.output LOCALVERSION= -j$(nproc)
+          make O=.output LOCALVERSION= -j$(nproc) Image.gz vmlinuz.efi
```

```suggestion
          make O=.output LOCALVERSION= -j$(nproc) Image.gz vmlinuz.efi modules
```
I think this should now also include "modules", so they're built in build phase and not in modules_install phase (as dependency for modules_install)